### ISSUE TITLE
Can't start ShB, just ARR

### ISSUE THREAD

**Issue by michanrt** [goaaats/FFXIVQuickLauncher#63]
Hello, 
Each time I start the game with the xivlauncher, I just have ARR. 
I remember to have a tab in the options to change my game version, but, I can't find it anymore since last launcher update, I guess (not really sure about it, sorry).
I tried the "Enable experimental UID cache", no change - Reset 
I deleted XIVLauncher.exe & XIVLauncher.exe.config, re-downloaded the latest version, cleaned %appdata% and searched for some entries in registry.

Launcher version ; V2.6.0.0 bdd1e1b
Windows 10 1809

screenshot of options attached

![xivlauncher_options](https://user-images.githubusercontent.com/49096836/60654705-cc12d480-9e4c-11e9-961e-2032c744df48.png)

Any idea ?

**Comment by goaaats** [comment]
And you definitely have the expansions on your account? If you log in with the official launcher, will it start with shadowbringers?

**Comment by michanrt** [comment]
Yes, I have the expansion on my account and can play it when using official launcher.

**Comment by goaaats** [comment]
Please mail me the output.log file from %AppData%\XIVLauncher to meeboo at protonmail com - I might have to give you a debug version to check this out

**Comment by michanrt** [comment]
It's a bit awkward, but it works again.
I have just downgrade the launcher to version 2.5, start it, it says that an update is available, downloaded the archive installed it, then after asking my OTP and, welcome screen is here \o/
I restarted the game, ShB again, restarted again to be sure, ShB !
I don't know what happened, but I notice something : 
- OTP was asked one time, then, I could restart the game without asking for a new OTP, I found it weird, but very convenient ^o^, but now, it asks me an OTP each time. Maybe the issue comes from this behavior, and what resolved it, I don't know. 
The file output.log is more verbose, I'm sending it.

Thanks goaaats for the work and tool you provide !
/cheer
